scipy/interpolate/_polyint.py

```diff
@@ -538,20 +538,13 @@ class BarycentricInterpolator(_Interpolator1DWithDerivatives):
     efficient changing of the y-values to be interpolated,
     and updating by adding more x- and y-values.
 
-    For reasons of numerical stability, this function does not compute
```

Comment:
Removed as this seems too vague to be useful

Reply:
Was a nice factoid IMO. Fine to remove though if you feel like it's extraneous.

Reply:
I'll move to the notes since it is an interesting point; it just didn't seem to be necessary to include it in the summary